clustering/weld/core/src/main/java/org/wildfly/clustering/weld/contexts/SerializableContextualInstanceImplMarshaller.java

```diff
@@ -78,7 +79,8 @@ public SerializableContextualInstanceImpl<C, I> readFrom(ProtoStreamReader reade
                 public I run() {
                     try {
                         return (I) instantiator.newInstance(targetClass);
-                    } catch (InstantiationException | IllegalAccessException e) {
+                    } catch (InstantiationException | IllegalAccessException | NoSuchMethodException |
```

Comment:
FTR, this is a side effect of changes to `DefaultProxyInstantiator` done as part of https://github.com/weld/core/commit/d744b62fe98952d0cfd998c25aad57849b3190c4